calcculator-app/src/components/Input.js

```diff
@@ -0,0 +1,277 @@
+import React from "react";
+import Grid from "./Grid.js";
+import Mod from "./Mod.js";
+import Equal from "./Equal.js";
+import Clear from "./Clear.js";
+import "./styles/Input.css";
+
+class Input extends React.Component {
+  state = {
+    input: [0],
+    firstDisplay: [],
+    mod: "",
+    secondDisplay: [],
+    answer: 0,
+    // equal:
+  };
+
+  handleInputChange = (event) => {
+    //let display = event.target.value.split("");
+  
```

Comment:
Looks good Farrah, but lets remember to remove debugger statements and commented out code for deployment